src/Tools/dotnet-monitor/Auth/ApiAuthenticationOptions.cs

```diff
@@ -10,7 +13,14 @@ namespace Microsoft.Diagnostics.Tools.Monitor
 {
     internal sealed class ApiAuthenticationOptions
     {
+        [Description("API Key in Hashed form. Each byte should be two hex based digits.")]
+        [RegularExpression("[0-9a-fA-F]+")]
+        [MinLength(64)]
```

Comment:
Looks wrong. Can't you specify exact length on the regex?

```suggestion
        [RegularExpression("[0-9a-fA-F]{64}")]
```

Reply:
Note there's no MaxLength. We want agility here, in case we want a longer hash key in the future.